### PR TITLE
[3.11] .NET Core: remove information about EOL versions

### PR DESCRIPTION
**Pull request by tmds**

### CODE REVIEW

**Comment by tmds** [comment]
@ahardin-rh can you review this, or suggest someone that can?

**Comment by tmds** [comment]
cc @Levi-Le @RheaAyase @aslicerh @omajid

File: using_images/s2i_images/dot_net_core.adoc [hunk 1]
```diff
@@ -33,61 +33,16 @@ and Python from within {product-title}.
...
-For release details related to .NET Core version 2.1, see
-link:https://access.redhat.com/documentation/en-us/net_core/2.1/html/release_notes_for_containers/[Release
-Notes for Containers].
-
-Versions 1.1 and 1.0 (`rh-dotnetcore11` and `rh-dotnetcore10`) ship with the
-*_project.json_* build system (`1.0.0-preview2` SDK). See the Known Issues
-chapter in the
-link:https://access.redhat.com/documentation/en/net-core/1.1/paged/release-notes/[version
-1.1 Release Notes] for details on installing this SDK on a non-RHEL system.
+The link:https://access.redhat.com/support/policy/updates/net-core[.NET Core Life Cycle] lists the currently supported versions of .NET Core.
```
Comment (omajid) [inline using_images/s2i_images/dot_net_core.adoc:36]: :heart:

Comment (aslicerh) [reply using_images/s2i_images/dot_net_core.adoc:36]: Seconded.

**Comment by aslicerh** [comment]
I do, personally, prefer this style of pointing people to a 'single source of truth', both in the supported versions page and the getting started guide.

Assuming that meets the vision/guidelines of the Docs team, I'm good with this.

File: using_images/s2i_images/dot_net_core.adoc [hunk 2]
```diff
@@ -212,70 +167,6 @@ If you ran a standard installation, the image stream will be present.
...
-----
-$ oc new-app --template dotnet-example
-----
-
-The link:https://github.com/aspnet/MusicStore[.NET Core MusicStore application]
-using PostgreSQL as database can be deployed with:
-
 ----
-$ oc new-app --template=dotnet-pgsql-persistent
+$ oc new-app dotnet:3.1~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-3.1 --context-dir app
```
Comment (vikram-redhat) [inline using_images/s2i_images/dot_net_core.adoc:171]: This part needs QE to sign off on.

@tmds do you know who from QE can do that?

Comment (RheaAyase) [reply using_images/s2i_images/dot_net_core.adoc:171]: _\*signs off*_

**Pull request merged**